toolkit/resources/imageconfigs/packagelists/iso-initrd-packages.json

```diff
@@ -65,13 +56,9 @@
         "shim",
         "sqlite",
         "systemd",
-        "tar",
         "tdnf",
```

Comment:
tdnf is needed to for OS installation from initrd image.

Reply:
Some background here: The initrd installer acts the same way our normal installer works for offline builds. It configures a 'fake' package repo locally, and tdnf can install packages from that local repo instead of using online repos.